extensions/ch_vd_sire/views/regulation_pression.sql

```diff
@@ -1,57 +1,30 @@
+CREATE VIEW qwat_ch_vd_sire.regulation_pression AS 
 	SELECT
-		id AS ID_Num
-		, qwat_ext_ch_vd_sire_remarque || remark AS Remarque
-		, precision_code_sire AS Precision_Geo
-		, fk_distributor AS ID_Distributeur
-		, qwat_ext_ch_vd_sire_etat_exploitation AS Etat_Exploitation
-		, year AS Annee_Construction
-		, NULL::text AS Nom_Descriptif
-		, folder_identification AS Numero_Dossier
-		, fk_pressurezone AS ID_Zone_Pression
-		, watertype_code_sire AS Type_Eau
-		, CASE
-		    WHEN installation_type = 'pump' THEN 2
-				ELSE 0
-			END AS Genre
-		, altitude as Altitude
-		, 1 as Etat_Connexion
-		, NULL AS ID_Distributeur_2
-		, NULL AS ID_Zone_Pression_2
-		, 0 AS Type_Regulation
-		, NULL AS Valeur_Consigne_Reg
-		, 2 AS Telecommande_Incendie
-		, 0 AS Centrale_Telecommande
-		, qwat_ext_ch_vd_sire_adesafecter AS A_Desaffecter_PDDE
-		, ST_Force2D(geometry) AS geometry
+		id AS "ID_Num",
+		qwat_ext_ch_vd_sire_remarque || remark AS "Remarque",
+		distributor_name AS "Nom_Distributeur",
+		pressurezone_name AS "Nom_Zone_Pression",
+		precision_code_sire AS "Precision_Geo",
+		fk_distributor AS "ID_Distributeur",
+		qwat_ext_ch_vd_sire_etat_exploitation AS "Etat_Exploitation",
+		year AS "Annee_Construction",
+		name AS "Nom_Descriptif",
+		folder_identification AS "Numero_Dossier",
+		fk_pressurezone AS "ID_Zone_Pression",
+		watertype_code_sire AS "Type_Eau",
+		CASE
+			WHEN installation_type = 'pump'::qwat_od.installation_type THEN 2
+			ELSE 0
+		END AS "Genre",
+		altitude AS "Altitude",
+		1 AS "Etat_Connexion",
+		NULL::text AS "ID_Distributeur_2",
+		NULL::text AS "ID_Zone_pression_2",
+		0 AS "Type_Regulation",
+		NULL::double precision AS "Valeur_Consigne_Reg",
+		2 AS "Telecommande_Incendie",
+		0 AS "Centrale_Telecommande",
+		qwat_ext_ch_vd_sire_adesafecter AS "A_Desaffecter_PDDE",
+		st_force2d(geometry) AS geometry
 	FROM qwat_od.vw_export_installation
-	WHERE
-		fk_pressurecontrol_type IN (2801,2802) -- réducteur, coupe-pression
-
-	UNION
-
-	SELECT
-		id AS ID_Num
-		, qwat_ext_ch_vd_sire_remarque || remark AS Remarque
-		, precision_code_sire AS Precision_Geo
-		, fk_distributor AS ID_Distributeur
-		, qwat_ext_ch_vd_sire_etat_exploitation AS Etat_Exploitation
-		, year AS Annee_Construction
-		, NULL::text AS Nom_Descriptif
-		, folder_identification AS Numero_Dossier
-		, fk_pressurezone AS ID_Zone_Pression
-		, 1 AS Type_Eau
-		, 1 AS Genre
-		, altitude as Altitude
-		, 1 as Etat_Connexion
-		, NULL AS ID_Distributeur_2
-		, NULL AS ID_Zone_pression_2
-		, 0 AS Type_Regulation
-		, NULL AS Valeur_Consigne_Reg
-		, 2 AS Telecommande_Incendie
-		, 0 AS Centrale_Telecommande
-		, qwat_ext_ch_vd_sire_adesafecter AS A_Desaffecter_PDDE
-    , ST_Force2D(geometry) AS geometry
-	FROM qwat_od.vw_export_valve
-	WHERE fk_valve_function = 6101  -- vanne régulation
-		;
+	WHERE (fk_pressurecontrol_type = ANY (ARRAY[2801, 2802])) AND (status_functional IS TRUE OR fk_status = 1306) AND fk_distributor = 1;
```

Comment:
+GRANT SELECT, REFERENCES, TRIGGER ON TABLE qwat_ch_vd_sire.captage TO qwat_viewer;
+GRANT ALL ON TABLE qwat_ch_vd_sire.captage TO qwat_user;
+GRANT ALL ON TABLE qwat_ch_vd_sire.captage TO qwat_manager;

(add Grants)